notebook/supermusr-plotting-frames.ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# These values are set here and used further down\n",
+    "filename = \"trace-Tue 26 Jul 10-57-38 BST 2022.h5\"\n",
+    "# plot_frame = True\n",
+    "frame = 400 # Expected if plot_frame is True, alternatively use a timestamp\n",
+    "timestamp = 1658826552.89 # Expected in seconds UTC (Note: the data takes into consideration nanoseconds) Example: 1658826552.89\n",
+    "detector_to_plot = 0 # The value of the detector to plot"
```

Comment:
It would be good to provide a list of channels to plot here.

A more general point, "channel" would be better terminology to use here.